engine/src/cmd/movable.cpp

```diff
@@ -5,9 +5,40 @@
 #include "unit_find.h"
 #include "star_system.h"
 #include "universe.h"
+#include "game_config.h"
 
 #include <iostream>
 
+static const float VELOCITY_MAX = GameConfig::GetVariable( "physics", "velocity_max", 10000);
+
+//for the heck of it.
+static const float humanwarprampuptime = GameConfig::GetVariable( "physics", "warprampuptime", 5);
+
+//for the heck of it.
+static const float compwarprampuptime = GameConfig::GetVariable( "physics", "computerwarprampuptime", 10);
+
+static const float warprampdowntime = GameConfig::GetVariable( "physics", "warprampdowntime", 0.5f);
+static const float WARPMEMORYEFFECT = GameConfig::GetVariable( "physics", "WarpMemoryEffect", 0.9f);
+static const float maxplayerrotationrate = GameConfig::GetVariable( "physics", "maxplayerrot", 24);
+static const float maxnonplayerrotationrate = GameConfig::GetVariable( "physics", "maxNPCrot", 360);
+static const float warpstretchcutoff = GameConfig::GetVariable( "graphics", "warp_stretch_cutoff", 500000) * GameConfig::GetVariable( "physics", "game_speed", 1);
+static const float warpstretchoutcutoff = GameConfig::GetVariable( "graphics", "warp_stretch_decel_cutoff", 500000) * GameConfig::GetVariable( "physics", "game_speed", 1);
+static const float sec = GameConfig::GetVariable( "graphics", "insys_jump_ani_second_ahead", 4) / (GameConfig::GetVariable( "physics", "game_speed", 1) * GameConfig::GetVariable( "physics", "game_accel", 1));
+static const float endsec = GameConfig::GetVariable( "graphics", "insys_jump_ani_second_ahead_end",0.03f) /( GameConfig::GetVariable( "physics", "game_speed",1.0f) *GameConfig::GetVariable( "physics", "game_accel", 1.0f) );
+
+
+//Pi^2
+
+static const float  warpMultiplierMin = GameConfig::GetVariable( "physics", "warpMultiplierMin", 9.86960440109f);
+
+//C
+static const float  warpMultiplierMax = GameConfig::GetVariable( "physics", "warpMultiplierMax", 300000000 );
+
+//Pi^2 * C
+static const float warpMaxEfVel = GameConfig::GetVariable( "physics", "warpMaxEfVel", 2960881320.0f);
```

Comment:
Because this is both `static` and `const`, it gets initialized before vegastrike.config is actually loaded. So it never actually reads its value from the config file. This is what is causing #335 , I'm pretty sure.